lib/hangman.rb

```diff
@@ -0,0 +1,54 @@
+class Word
+
+  attr_reader :incorrect_guesses, :correct_guesses, :user_guesses
+
+  def initialize(word)
+    @word = word
+    # @user_guesses = [] # not needed?
+    @incorrect_guesses = []
+    @correct_guesses = ["_"] * @word.length
```

Comment:
I think this could have a different name, I took a bit of a double take at `correct_guesses` that is actually an array of strings that are `_`! Took me a minute to realise this gets replaced! 

Naming is hard though, maybe something like `letters_in_word` - even that's not a great name though!

Reply:
Thanks for the tip :)